### DISCUSSION TITLE
How to hide hotkey's info from desktop?

### DISCUSSION THREAD

**Discussion by eluz1ve** [Manjaro-Sway/manjaro-sway#324] [Q&A] [answered]
Hi, recently I install Manjaro Sway Edition, read alot config files, but can't find how to hide hotkey's info from desktop (wallpaper). Help please.

**Comment by boredland** [comment, marked as answer]
does [this](https://github.com/manjaro-sway/manjaro-sway/blob/main/SUPPORT.md#how-can-i-disable-the-help-onscreen-menu-permanently) still work? or do you want to hide it temporarily?
![image](https://user-images.githubusercontent.com/4662748/190398982-53a7d3d1-2ab3-4458-a24a-699a3449341c.png)

**Reply by boredland** [reply]
If you'd like to have it on another key, [this](https://github.com/manjaro-sway/desktop-settings/blob/e143f07a9f469430e6d99e9445598c9d4c6429d0/community/sway/etc/sway/modes/default#L153) is how it is being called in sway config: `$bindsym $mod+question exec $help`

**Reply by eluz1ve** [reply]
Thanks, that's what i've looked for!

**Reply by robin-thoene** [reply]
Hey together, I stumbled over this and I am currently searching for a way to hide the menu automatically after boot.

The expected behavior I want to achieve is to not display the help menu after boot, but still have the possibility to toggle it using the default shortcut (mod+question).

I customized a lot of things already but I struggle to find the correct place to implement this.
